src/App.css

```diff
@@ -244,7 +237,37 @@ input {
 
 .BackButton_List:hover {
   transform: scale(1.2);
-  transition: .1s;
+  transition: 0.1s;
   transition-timing-function: ease-in;
+}
 
+.duplicateMessage {
+  position: fixed;
+  top: 15%;
+  left: 50%;
+  -webkit-transform: translate(-50%, -50%);
+  transform: translate(-50%, -50%);
+  height: fit-content;
+  width: fit-content;
+  padding: 10px;
+  margin: auto;
+  border-radius: 20px;
+  background: #ffd800;
+  animation: slide 2.5s;
+  opacity: 0.7;
+}
+
+@keyframes slide {
```

Comment:
Nice animated stuff. I dig it.

Reply:
I second that! 

![bitmoji](https://render.bitstrips.com/v2/cpanel/0c62d877-a0fb-4266-89d5-4d80ec96324f-141935a4-509e-461c-bba3-4228c119c4d3-v1.png?transparent=1&palette=1&width=100)

Reply:
Agreed! Very nice!

Reply:
👏